### ISSUE TITLE
Cannot confirm seed phrase due to duplicate seed words

### ISSUE THREAD

**Issue by frankiecee** [MetaMask/metamask-mobile#1233]
When going through the flow of backing up and confirming the wallet seed phrase,  it is not possible to complete the confirmation if there are two of the same seed word. (see screenshot).

![Screenshot_2019-11-04-22-56-05-534_io metamask](https://user-images.githubusercontent.com/11490430/69970025-80dc1b00-14eb-11ea-9ee0-2ef137248537.png)

It looks like when one of the duplicate seed words is selected, both of them are greyed out and cannot be used again. I haven't tried to reproduce this issue since it is contingent on that any newly created wallet has duplicate seed words.

support case for reference: 26406

**Comment by brunobar79** [comment]
duplicate of #1020
